holmes/plugins/toolsets/aks-node-health.yaml

```diff
@@ -47,3 +48,18 @@ toolsets:
         user_description: "check inbound network connectivity for AKS cluster {{ CLUSTER_NAME }} in resource group {{ RESOURCE_GROUP }}"
         command: |
           az aks check-network inbound --name {{ CLUSTER_NAME }} --resource-group {{ RESOURCE_GROUP }}
+      - name: "list_vmss_names"
+        description: |
+          Lists all VMSS names in the cluster node resource group.
+          Prerequisites: get_node_resource_group
+        user_description: "lists all VMSS names in {{ NODE_RESOURCE_GROUP }}"
+        command: |
+          az vmss list -g {{ NODE_RESOURCE_GROUP }} --query '[*].name' -o tsv --only-show-errors
+      - name: "vmss_run_command"
+        description: |
+          Execute a shell command on a specific VMSS VM instance using az vmss run-command.
+          VM_ID is the instance ID of the VMSS, which can be derived from node names.
+          Prerequisites: get_node_resource_group, list_vmss_names
```

Comment:
Why do we need to call the `get_node_resource_group` here?

Reply:
Because aks host all cluster resources like vms in another resource group called node resource group.